src/EnergyPlus/General.cc

```diff
@@ -1101,21 +1036,17 @@ void ScanForReports(EnergyPlusData &state,
     // First time routine is called, all the viable combinations/settings for the reports are
     // stored in SAVEd variables.  Later callings will retrieve those.
 
-    // Using/Aliasing
-
-    // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-    int NumReports;
-    int RepNum;
-    int NumNames;
-    int NumNumbers;
-    int IOStat;
-    auto &cCurrentModuleObject = state.dataIPShortCut->cCurrentModuleObject;
-
```

Comment:
[src/EnergyPlus/General.cc:1107]:(style),[variableScope],The scope of the variable 'NumReports' can be reduced.
[src/EnergyPlus/General.cc:1108]:(style),[variableScope],The scope of the variable 'RepNum' can be reduced.
[src/EnergyPlus/General.cc:1109]:(style),[variableScope],The scope of the variable 'NumNames' can be reduced.
[src/EnergyPlus/General.cc:1110]:(style),[variableScope],The scope of the variable 'NumNumbers' can be reduced.
[src/EnergyPlus/General.cc:1111]:(style),[variableScope],The scope of the variable 'IOStat' can be reduced.
[src/EnergyPlus/General.cc:1112]:(style),[variableScope],The scope of the variable 'cCurrentModuleObject' can be reduced.